en/docs/learn/api-controller/ci-cd-with-wso2-api-management.md

```diff
@@ -207,16 +207,101 @@ For more information on initializing an API Project using OpenAPI/Swagger Specif
+The **apictl** tool should be installed in the automation servers to begin the process. As the tool supports a variety of platforms, including Linux/Windows and macOS, this can be done easily. 
 
-1.  Import the `SwaggerPetstore` API into the production environment and test the API by running the following sample command.
+#### a. Promoting API project(s) in a GIT repository using a unified approach
+
+The repository (Version Control System) needs to be cloned into the instance that is executing the CI/CD process. From 3.2.0, **apictl** has the inbuilt support to integrate with a GIT based version control system. It gives a unified command `vcs deploy` to deploy any type of project (APIs, API Products and Apps).
+
+1.  Change your working directory (`cd`) into the cloned git repository.
+2.  Run `vcs status` command to see the available changes that needs to be deployed to the production environment.
+
+    !!! example
+        ```bash
+        $ apictl vcs status -e prod
+        Projects to Deploy (1)
+
+        APIs (1) ...
+        1: [save]		SwaggerPetstore-1.0.0: (SwaggerPetstore-1.0.0)
+        ```
+
+3.  Import the **SwaggerPetstore** API into the production environment by running the following sample command.
 
     !!! tip
         - Make sure you have already logged-in to the `prod` environment. For more information, see 
         [Login to an Environment]({{base_path}}/learn/api-controller/getting-started-with-wso2-api-controller/#login-to-an-environment).
 
         - A user with `admin` role is allowed to import APIs. To create a custom user who can import APIs, refer [Steps to Create a Custom User who can Perform API Controller Operations]({{base_path}}/learn/api-controller/advanced-topics/creating-custom-users-to-perform-api-controller-operations/#steps-to-create-a-custom-user-who-can-perform-api-controller-operations).
 
+    !!! example
+        ```bash
+        $ apictl vcs deploy -e prod
+        Deploying Projects (1)...
+
+        APIs (1) ...
+        1: SwaggerPetstore-1.0.0: (SwaggerPetstore-1.0.0)
+        Successfully imported API
+        ```
+
+    Now the tool will automatically detect the target environment and prepare a new artifact containing environment-related details. 
+
+    If you run `vcs deploy` command again, you will see the below output indicating that the deployment is already up-to-date.
+
+    !!! example
+        ```bash
+        $ apictl vcs deploy -e prod
+        Everything is up-to-date
+        ```
+
+    !!! tip
+        If you have multiple APIs to promote through CI/CD, you can commit those API projects as well to same repository. The rest of the process will be the same as above.
+ 
+        If you have export (or initiallized) and commited another API **Pizzashack-1.0.0** to the same repository and then run `vcs status` and `vcs deploy` commands, you will be able to see the below outputs. 
```

Comment:
Can we change this as If you have export (or initialized) and committed another API **Pizzashack-1.0.0** to the same repository and then run `vcs status` and `vcs deploy` commands, you will be able to see the below outputs. Since this has some typos.

Reply:
It's the initialized word, right? Is there any other typo?

Reply:
And "committed" also

Reply:
Ah, correct.

Reply:
Fixed